networkx/drawing/layout.py

```diff
@@ -983,6 +983,13 @@ def spiral_layout(G, scale=1, center=None, dim=2, resolution=0.35, equidistant=F
     --------
     >>> G = nx.path_graph(4)
     >>> pos = nx.spiral_layout(G)
+    >>> nx.draw(G, pos=pos)
+
+    References
+    ----------
+    .. [1] Cheong, Se-Hang, and Yain-Whar Si. "Force-directed algorithms for schematic drawings and
+        placement: A survey." Information Visualization 19, no. 1 (2020): 65-91.
+        https://doi.org/10.1177%2F1473871618821740
```

Comment:
I'm a bit confused - I just looked at the arXiv paper referenced here and didn't see anything about spiral layouts... are we certain that this reference actually mentions this layout algorithm?

Reply:
Hmmm... Was this intended to go into a Force layout function like Fruchterman–Reingold or Force-Atlas-2?
Maybe spring_layout and spiral_layout got confused?

Reply:
Oops, so sorry, it's now that I am going through my research and indeed I mistook the spring layout for `spiral layout`.
Please sir, is the reference suitable to use for `spring layout`?

Reply:
Please sir, the issue also stated that references of similar layout in other graph visualization.
I wish to know if this link is appropriate for it 
https://rdrr.io/bioc/netbiov/man/plot.spiral.graph.html

Reply:
> Please sir, the issue also stated that references of similar layout in other graph visualization.

Thanks for looking into it! I don't think this version of the spiral layout is directly related to the one in NetworkX. Of course some details appear similar, but I don't think it's appropriate as a reference for `nx.spiral_layout`.

Reply:
Okay sir, thanks.